<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmInstalarJuego</class>
 <widget class="QDialog" name="frmInstalarJuego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Instalar un juego</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/gr-lida.qrc">
    <normaloff>:/img16/applications.png</normaloff>:/img16/applications.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gBoxMount">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Montaje de ISO, CUE / BIN, etc</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_virtualdrive">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_Datos_43">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Unidad virtual a usar:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbx_virtual_drive">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_Datos_42">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Imagenes:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QListWidget" name="lw_vd_images">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="hSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>163</width>
            <height>27</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_virtual_drive_montar">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Montar</string>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/virtual_drive.png</normaloff>:/img16/virtual_drive.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QFrame" name="frame_btns_vd">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_virtual_drive_edit">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/editar.png</normaloff>:/img16/editar.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btn_vd_image_list_add">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/list_add.png</normaloff>:/img16/list_add.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btn_vd_image_list_clear">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btn_vd_image_list_subir">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/go-up.png</normaloff>:/img16/go-up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btn_vd_image_list_delete">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/list_remove.png</normaloff>:/img16/list_remove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btn_vd_image_list_bajar">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/go-down.png</normaloff>:/img16/go-down.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gBoxDbx">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Usando DOSBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_dosbox" rowstretch="0,0,0,0,0,0">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnDestino">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/carpeta_0.png</normaloff>:/img16/carpeta_0.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QFrame" name="frame_list_mount">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_list_mount">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnDeleteIso">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/list_remove.png</normaloff>:/img16/list_remove.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnSubirIso">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/go-up.png</normaloff>:/img16/go-up.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnDirFile_clear">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnDirFile">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/list_add.png</normaloff>:/img16/list_add.png</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btnBajarIso">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../res/gr-lida.qrc">
             <normaloff>:/img16/go-down.png</normaloff>:/img16/go-down.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0" rowspan="3">
          <widget class="QListWidget" name="lw_MultiIso">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="txtMontaje_path">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>24</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Seccionar origen:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lb_Dbx_75">
        <property name="text">
         <string>DOSBox a usar:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLabel" name="lb_Montaje_1">
        <property name="text">
         <string>Seccionar la unidad, carpeta o CD-Rom que desea montar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbxDbx_EmuKey">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seccionar destino:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtDestinoPath">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbxMontaje_type_drive">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Tipo de origen:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btnDestinoClear">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>26</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/gr-lida.qrc">
          <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <layout class="QHBoxLayout" name="hLayout">
        <item>
         <spacer name="hSpacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>163</width>
            <height>27</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnEjecutarDbx">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>Ejecutar DOSBox</string>
          </property>
          <property name="icon">
           <iconset resource="../res/gr-lida.qrc">
            <normaloff>:/img16/cat/dosbox.png</normaloff>:/img16/cat/dosbox.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>97</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/gr-lida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_vd_image_list_add</tabstop>
  <tabstop>btn_vd_image_list_clear</tabstop>
  <tabstop>cbx_virtual_drive</tabstop>
  <tabstop>btn_virtual_drive_montar</tabstop>
  <tabstop>cbxDbx_EmuKey</tabstop>
  <tabstop>cbxMontaje_type_drive</tabstop>
  <tabstop>btnDirFile</tabstop>
  <tabstop>txtMontaje_path</tabstop>
  <tabstop>btnDirFile_clear</tabstop>
  <tabstop>lw_MultiIso</tabstop>
  <tabstop>btnSubirIso</tabstop>
  <tabstop>btnBajarIso</tabstop>
  <tabstop>btnDeleteIso</tabstop>
  <tabstop>btnDestino</tabstop>
  <tabstop>txtDestinoPath</tabstop>
  <tabstop>btnDestinoClear</tabstop>
  <tabstop>btnEjecutarDbx</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/gr-lida.qrc"/>
 </resources>
 <connections/>
</ui>
